README.md

```diff
@@ -10,24 +10,27 @@ The project is in the early development stage. More description will be added la
 
 Google Python Style Guide https://google.github.io/styleguide/pyguide.html
 
-### Installing dependencies
+### Installation
 
    This project depends on numpy apache-beam pyspark absl-py dataclasses
  
-   For installing with pip please run: 
+   For installing with poetry please run: 
```

Comment:
Please leave both options with pip and with poetry

Reply:
Yeah, changed it. Please check